training_event/views.py

```diff
@@ -119,3 +119,14 @@ def list(self, request):
                 training_event.models.EventCoefficient.ROUND_CHOICES)
         ]
         return Response(round_choices, status=status.HTTP_200_OK)
+
+
+class EventCoefficientRoleChoices(viewsets.ViewSet):
+    '''Create API view for get role choices of event coefficient.'''
+    def list(self, request):
+        '''define how to get role choices.'''
+        role_choices = [
+            item[1] for item in (
```

Comment:
Consider returning with id, so that you can return id back later instead of returning role name to backend server.